src/server/services/songSubmissionService.ts

```diff
@@ -0,0 +1,52 @@
+import client from "../config/esConnection";
+import { Song } from "../schemas/Song";
+import { SongSubmission } from "../schemas/SongSubmission";
+import { User } from "../schemas/User";
+import {v4 as uuid} from 'uuid';
+
+
+const getSongSubmissions = async () : Promise<SongSubmission[]> => {
+    const {body} = await client.search({
+        index: "songsubmissions",
```

Comment:
nitpick: rename `songsubmissions` to `songSubmissions`

Reply:
indices have to be all lowercase :/

Reply:
oh rip